physics/module_mp_nssl_2mom.F90

```diff
@@ -908,12 +956,13 @@ MODULE module_mp_nssl_2mom
       real :: cp = 1004.0, rd = 287.04
       real :: rw = 461.5              ! gas const. for water vapor
       REAL, PRIVATE ::      cpl = 4190.0
-     REAL, PRIVATE ::      cpigb = 2106.0
+      REAL, PRIVATE ::      cpigb = 2106.0
       real :: cpi 
       real :: cap 
       real :: tfrcbw
       real :: tfrcbi
       real :: rovcp
+      real, public :: rdorv = 0.622
```

Comment:
I realize that there are still several physical constants being redefined in this module (which can introduce inconsistencies between this scheme and the host and other CCPP schemes), but is rdorv being defined here when it is also set during the init phase? Just in case the constant intialization subroutine is skipped?

Reply:
Good question. I think that might be the only constant I added for CCPP the first time. It is only used in the driver routine (and only in the CCPP version -- not used in WRF). I'm not sure why I set it to public... probably should be private. I see there are a couple other constants that don't have initial values, and others that do, so that's not consistent. Would you prefer that all have initial values or none?

Reply:
If this same code is being run in a situation where the physical constants are not being set from the host (non-CCPP) AND in a situation where they are, then providing default initial values would be prudent. Otherwise, it may be misleading if someone is reading the code? Probably not a huge deal.

Reply:
I might as well set all the values for consistency while I'm making other minor changes.